packages/docs/pages/reference/elements/problem.mdx

```diff
@@ -8,12 +8,50 @@ import {
 
 # `<problem>`
 
-Short description of what the `<problem>` tag does.
+The `<example>`, `problem`, and `question` tags are closely related and have
+the same syntax.  The `title` element is optional but encouraged, and the
+structure can contain `hint`s, `answer`s, and `solution`s (in that order).
 
 ## Syntax
 
 <SyntaxDisplay name="problem" />
 
 ## Examples
 
-    
\ No newline at end of file
+ ```ptx-example
+<FRAGMENT parents="p">
+<problem>
+  <title>Row and column spaces</title>
+  <p>
+  We know that the rank of a matrix <m>A</m> and of its row-reduced echelon
+  form <m>R</m> is the same.  Determine if the row space and column space of
+  <m>A</m> and <m>R</m> are always the same.
+  </p>
```

Comment:
Interesting. I think this is not good form. You either use only `<p>`s or you use `<statement>`, `<hint>`, etc.